### PR TITLE
[documentation] upd tcp and kafka plugin docs

### PR DESCRIPTION
**Pull request by brunooliveiramac**
Add documentation for:
- Threaded option for Kafka input plugin
- Workers option for Kafka output plugin

### CODE REVIEW

**Review by tonyschak — APPROVED** [review]

**Comment by brunooliveiramac** [comment]
@patrick-stephens could you check if that is enough :D

File: pipeline/inputs/tcp.md [hunk 1]
```diff
@@ -7,14 +7,15 @@ The **tcp** input plugin allows to retrieve structured JSON or raw messages over
 The plugin supports the following configuration parameters:
 
 | Key          | Description                                                                                                                                                                                                                                                    | Default |
```
Comment (cnorris-cs) [inline pipeline/inputs/tcp.md:9]: Let's start getting these table formats in line.

```suggestion
| Key          | Description | Default |
```

File: pipeline/inputs/tcp.md [hunk 2]
```diff
@@ -7,14 +7,15 @@ The **tcp** input plugin allows to retrieve structured JSON or raw messages over
...
-| ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| ------------ |----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
```
Comment (cnorris-cs) [inline pipeline/inputs/tcp.md:10]: This formatting aligns with the suggestion for the previous line.

```suggestion
| ------------ | ----------- | ------- |
```

File: pipeline/inputs/tcp.md [hunk 3]
```diff
@@ -7,14 +7,15 @@ The **tcp** input plugin allows to retrieve structured JSON or raw messages over
...
 | Format       | Specify the expected payload format. It support the options _json_ and _none_. When using _json_, it expects JSON maps, when is set to _none_, it will split every record using the defined _Separator_ (option below).                                        | json    |
 | Separator    | When the expected _Format_ is set to _none_, Fluent Bit needs a separator string to split the records. By default it uses the breakline character  (LF or 0x10).                                                                                               |         |
 | Source\_Address\_Key| Specify the key where the source address will be injected.                                                                                                                                                                                              |         |
+| Threaded     | Enhance the performance of data ingestion by allowing Fluent Bit to handle incoming data in parallel across multiple dedicated threads.                                                                                                                        | off     |
```
Comment (cnorris-cs) [inline pipeline/inputs/tcp.md:18]:
```suggestion
| Threaded     | Enhance the performance of data ingestion by letting Fluent Bit handle incoming data in parallel across multiple dedicated threads. | off |
```

**Review by cnorris-cs — APPROVED** [review]
This looks good from a technical writing perspective, and I'll add my approval based on that. This shouldn't replace any technical review of the content, but formatting-wise it looks good. Thanks for making those changes!

**Review by tonyschak — APPROVED** [review]

**Review by patrick-stephens — APPROVED** [review]

**Pull request merged**